bin/install_python_versions

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require_relative "./python_versions"
```

Comment:
Nit: this is the gnarliest bit to me - the script is useless without the specific `COPY` instructions to stitch these files together in the same path.

Could this use the copy of `python_versions.rb` that we already [copied in](https://github.com/dependabot/dependabot-core/pull/2587/files#diff-5dc00173d2accff302746c5a02269225R47)?
`CODE_DIR` should be available, but the I'm not sure of the ruby incantation to suggest.

Reply:
Yeah good point, I tried getting that version that was copied in, but all my attempts failed. But you're right and I'll try to think of a better solution for this 👍